lib/routes.json

```diff
@@ -1861,9 +1927,6 @@
           "required": true,
           "type": "string"
         },
-        "recursive": {
-          "type": "boolean"
-        },
```

Comment:
this is correct, because `recursive` does not accept true/false but only `1` as value